app/Config/CORS.php

```diff
@@ -0,0 +1,36 @@
+<?php namespace Config;
+
+use CodeIgniter\Config\BaseConfig;
+
+/**
+ * Class CORS
+ *
+ * Stores the default settings for the CORS, if you choose to use it. The
+ * values here will be read in and set as defaults for the site. If needed,
+ * they can be overridden on a page-by-page basis.
+ *
+ * Suggested reference for explanations:
+ *   - https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
+ *   - https://www.php.net/manual/en/function.header.php
+ *
+ * @author Marcelo Ratton <www.MarceloRatton.com>
+ * @package Config
```

Comment:
```suggestion
 *
```
While the intention is noble, the coding style guide does not accept @author tags. Also, @package tags are unnecessary since the file is namespaced.

Reply:
other system files have these tags, so keep it

Reply:
According to the PHPDoc documentation, @package tags are only required for **non-namespaced** classes. The use of package tags was carried over from CI3. Since CI4 is actively using namespaces, the tag is really not necessary. Those lingering package tags will be addressed down the road, so please do not add more.